spec/workers/supplejack_api/index_remaining_records_in_queue_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+module SupplejackApi
+  describe IndexRemainingRecordsInQueue do
+    let(:index_remaining_records_in_queue) { SupplejackApi::IndexRemainingRecordsInQueue.new }
```

Comment:
Similarly, isn't this the subject from the `describe` and implicitly initialised?